models/staging/vocabulary/_vocabulary__sources.yml

```diff
@@ -2,7 +2,7 @@ version: 2
 
 sources:
   - name: vocabulary
-    schema: "{% if var('seed_source', false) %}{{ target.schema }}_vocab_seeds{% else %}{{ target.schema }}{% endif %}"
+    schema: "{% if var('seed_source', false) %}{{ target.schema }}_vocab_seeds{% else %}{{ target.schema }}_vocab{% endif %}"
```

Comment:
this sequesters the source vocab tables in their own schema, before they were going in the target schema and getting overwritten by the final OMOP tables